oneanddone/tests/functional/conftest.py

```diff
@@ -71,3 +76,10 @@ def task(base_url, is_local):
     if is_local:
         from oneanddone.tasks.tests import TaskFactory
         return TaskFactory.create()
+
+
+@pytest.fixture(scope='session')
+def nonrepeatable_task(base_url, is_local2):
```

Comment:
This fixture does not use `base_url` so we can remove it. Also, can we rename it to `non_repeatable_task`?